### DISCUSSION TITLE
Shortcut for dev tool?

### DISCUSSION THREAD

**Discussion by dev-song** [responsively-org/responsively-app#811] [Q&A] [answered]
I'm using macOS Ventura 13.1... (Intel Mac)
Is there a shortcut for `Inspect Element` or opening a dev tool?
I'd be much better if I can check elements without moving mouse cursor to top-right of the app everytime...

**Comment by manojVivek** [comment, marked as answer]
Have opened an issue for this here: https://github.com/responsively-org/responsively-app/issues/812
